templates/ts-fastify-mongodb-backend/src/index.ts

```diff
@@ -35,16 +37,13 @@ async function start() {
     dataProviderCreator: createMongoDbProvider(db)
   });
 
-  const apolloServer = new ApolloServer({
-    typeDefs,
-    resolvers: [resolvers, noteResolvers],
-    context: contextCreator
-  });
-
-  apolloServer.applyMiddleware({ app });
+  app.register((mercurius as any), { 
```

Comment:
I think this kinda ended up being very complex PR for community member. 
We are missing subscription support. See: https://github.com/mercurius-js/mercurius